calico/networking/configuring/add-maglev-load-balancing.mdx

```diff
@@ -0,0 +1,34 @@
+---
+description: Add Maglev load balancing to a Kubernetes service.
+---
+
+# Add Maglev load balancing to a service
+
+This guide shows you how to enable Maglev-style load balancing to a service.
+Maglev's consistent-hashing load balancing algorithm minimizes the amount of dropped packets and improves perfomance.
+This is especially useful for maintaining sessions during a load balancer failover.
```

Comment:
This isn't expecially useful, it is the point! This intro should say that maglev loadbalancing reduces disruptions during LBs outage by failing over to other LBs as all LBs are making consisten backend selection without mutual synchronization. It is only used balancing ingress from outside the cluster. All nodes can serve as the LBs. etc.